src/components/App/App.js

```diff
@@ -1,23 +1,32 @@
 import './App.css';
 import React, {Component} from 'react'
 import MovieContainer from '../MovieContainer/MovieContainer'
-import movieData from "../../movieMock.js"
+import { fetchAllMovies } from '../../apiCalls';
 import MovieInfo from '../MovieInfo/MovieInfo';
 import FooterForm from '../FooterForm/FooterForm';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData,
-      selectedMovie : null
+      allMovies: [],
+      selectedMovie : 0,
+      error : null
     }
   }
 
   clickMovieSelect = (id) => {
-    const selectedMovie = this.state.allMovies.movies.find(movie => movie.id === id)
-    this.setState({selectedMovie})
-    console.log(selectedMovie.id)
+    this.setState({selectedMovie: id})
+    // const selectedMovie = this.state.allMovies.movies.find(movie => movie.id === id)
+    // this.setState({selectedMovie})
+    // console.log(selectedMovie.id)
+  }
+
+  componentDidMount() {
+    fetchAllMovies()
+    .then(data =>{
+      this.setState({allMovies : data.movies})
+    })  
```

Comment:
You'll want to add a .catch block here to respond to any errors. Think about what you want to actually have happen in the .catch here. Console logging is helpful to developers as they are building the code but wouldn't actually help the user if they were trying to use the application and the fetch failed for some reason.